saw-central/src/SAWCentral/Value.hs

```diff
@@ -52,15 +52,19 @@ module SAWCentral.Value (
     -- used by SAWCentral.Builtins, SAWScript.Interpreter
     evaluateTypedTerm,
     -- used by SAWScript.Interpreter (and in LocalEnv)
-    LocalBinding(..),
+--    LocalBinding(..),
     -- used by SAWScript.Interpreter, and appears in TopLevelRO
-    LocalEnv,
+--    LocalEnv,
     -- used by SAWScript.Interpreter, and in mergeLocalEnv
-    addTypedef,
+--    addTypedef,
     -- used by SAWScript.REPL.Monad, and by getMergedEnv
-    mergeLocalEnv,
+--    mergeLocalEnv,
     -- used by SAWCentral.Builtins, SAWScript.Interpreter, and by getCryptolEnv
-    getMergedEnv, getMergedEnv',
+--    getMergedEnv, getMergedEnv',
+    -- used by TBD XXX
```

Comment:
Surely this _is_ used somewhere? Worth filling in this comment saying where exactly. (Similarly for the other "used by TBD XXX" comments below.)